### PR TITLE
Add informs news

### PR DESCRIPTION
**Pull request by ZedongPeng**
- Add informs news
- Fix the LatinX bug

### CODE REVIEW

File: _posts/2023-10-18-INFORMS.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: 2023 INFORMS Annual Meeting
+description: '@Phoenix'
+image: assets/images/posts/INFORMS-2023-David-Zedong.jpeg
+---
+
+We wrap up our first INFORMS annual meeting.
```
Comment (bernalde) [inline _posts/2023-10-18-INFORMS.md:8]:
```suggestion
We wrapped up our first INFORMS annual meeting. 
```

File: _posts/2023-10-18-INFORMS.md [hunk 2]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: 2023 INFORMS Annual Meeting
+description: '@Phoenix'
+image: assets/images/posts/INFORMS-2023-David-Zedong.jpeg
+---
+
+We wrap up our first INFORMS annual meeting. 
+- [Zedong Peng](https://SECQUOIA.github.io/2-members.html) presenting the work on open-source discrete nonlinear optimizer [**MindtPy**](https://pyomo.readthedocs.io/en/stable/contributed_packages/mindtpy.html).
```
Comment (bernalde) [inline _posts/2023-10-18-INFORMS.md:9]:
```suggestion
- [Zedong Peng](https://SECQUOIA.github.io/2-members.html) presented the work on open-source discrete nonlinear optimizer [**MindtPy**](https://pyomo.readthedocs.io/en/stable/contributed_packages/mindtpy.html). 
```

File: _posts/2023-10-18-INFORMS.md [hunk 3]
```diff
@@ -0,0 +1,18 @@
+---
+layout: post
+title: 2023 INFORMS Annual Meeting
+description: '@Phoenix'
+image: assets/images/posts/INFORMS-2023-David-Zedong.jpeg
+---
+
+We wrap up our first INFORMS annual meeting. 
+- [Zedong Peng](https://SECQUOIA.github.io/2-members.html) presenting the work on open-source discrete nonlinear optimizer [**MindtPy**](https://pyomo.readthedocs.io/en/stable/contributed_packages/mindtpy.html). 
+- [David Esteban Bernal Neira](https://secquoia.github.io/1-bernalde.html) chairing the session on **hybrid classical quantum algorithms for optimization** !
```
Comment (bernalde) [inline _posts/2023-10-18-INFORMS.md:10]:
```suggestion
- [David Esteban Bernal Neira](https://secquoia.github.io/1-bernalde.html) chaired the session on **hybrid classical quantum algorithms for optimization** !
```
and presented as well.

**Review by bernalde — CHANGES_REQUESTED** [review]

**Comment by ZedongPeng** [comment]
The post has been modified according to the comments. @bernalde

**Pull request merged**